NineChronicles.DataProvider/Store/Models/PetEnhancementModel.cs

```diff
@@ -0,0 +1,39 @@
+namespace NineChronicles.DataProvider.Store.Models
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+
+    public class PetEnhancementModel
+    {
+        [Key]
+        public string? Id { get; set; }
+
+        public long BlockIndex { get; set; }
+
+        public string? AgentAddress { get; set; }
+
+        public AgentModel? Agent { get; set; }
+
+        public string? AvatarAddress { get; set; }
+
+        public AvatarModel? Avatar { get; set; }
+
+        public int PetId { get; set; }
+
+        public int PreviousPetLevel { get; set; }
+
+        public int TargetLevel { get; set; }
+
+        public int OutputPetLevel { get; set; }
+
+        public int ChangedLevel { get; set; }
+
+        public decimal BurntNCG { get; set; }
+
+        public decimal BurntSoulStone { get; set; }
+
+        public DateTimeOffset Date { get; set; }
```

Comment:
```suggestion
        public DateOnly Date { get; set; }
```

Could we change this type to `DateOnly`? 🙏